doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -675,6 +675,10 @@ Wi-Fi samples
   * Added ``Auto-Security-Personal`` mode to the ``connect`` command.
   * Added support for the ``WPA-PSK`` security mode to the ``wifi_mgmt_ext`` library.
 
+* :ref:`wifi_promiscuous_sample` sample:
+
+  * Added the :ref:`wifi_promiscuous_sample` sample that demonstrates how to set promiscuous mode, establish a connection to an Access Point (AP), analyze incoming Wi-Fi packets, and print packet statistics.
```

Comment:
Since it's a new sample, first bullet is not required.
```suggestion
* Added the :ref:`wifi_promiscuous_sample` sample that demonstrates how to set Promiscuous mode, establish a connection to an Access Point (AP), analyze incoming Wi-Fi packets, and print packet statistics.
```